packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/ai/actions/subscribe.tsx

```diff
@@ -74,7 +74,12 @@ export const AISubscribe = ({
       onBeforeCheckout={onBeforeCheckout}
       checkoutOptions={checkoutOptions}
       renderer={props => (
-        <Button variant="primary" {...props} {...btnProps}>
+        <Button
+          variant="primary"
+          {...props}
+          {...btnProps}
+          data-testid="ai-subscribe-button"
```

Comment:
no test?

Reply:
Added a test, but seems pricing request not working in CI? So I removed.

Reply:
payment service is not enabled in CI, I will configure it later